src/components/fields/NumericOrDate.js

```diff
@@ -14,7 +14,7 @@ export class UnconnectedNumericOrDate extends Component {
       (isDateTime(this.props.fullValue) || isJSDate(this.props.fullValue));
 
     return fullValueIsDate ? (
-      <UnconnectedText {...this.props} placeholder={'yyyy-mm-dd 00:00:00.00'} />
+      <UnconnectedDateTimePicker {...this.props} placeholder={'yyyy-mm-dd 00:00:00.00'} />
```

Comment:
the placeholder should be `YYYY-MM-DD hh:mm:ss.xxx` or something like that, not a mix of letters and numbers :)